### DISCUSSION TITLE
clickbait detection

### DISCUSSION THREAD

**Discussion by SoniEx2** [dhowe/AdNauseam#2498] [Ideas]
it'd be extra nice if adnauseam could run an algorithm to detect how clickbaity an ad is before trying to click it, to maximize clicks towards clickbaity ads.

because apparently ad networks already filter out adnauseam clicks from the real user's clicks, so adding adnauseam doesn't actually do anything to poison your ad profile and thus doesn't do anything to prevent tracking.

(the assumption here is that clickbaity ads are more likely to get organic clicks so adnauseam clicks mask well within the volume of clickbaity ad clicks. this may not actually hold true in practice tho, it's not like adnauseam knows what kinda ads ppl click.)

**Comment by dhowe** [comment]
> an algorithm to detect how clickbaity an ad is

do you have an idea for such an algorithm?

**Reply by SoniEx2** [reply]
image recognition/ML/"AI"

**Reply by LilacPeregrine** [reply]
Is this what you were thinking of, or was it more on the side of clickbait news stories? (The linked github is to a dataset on clickbaity ads)
(https://github.com/eric-zeng/conpro-bad-ads-data)

**Comment by mneunomne** [comment]
Only way I can imagine this being implemented would be to have a list of domains which are prone to be clickbait ads, and have that as a "filter" for ad-clicking, or at least "higher-frequency-ad-clicking".
